### PR TITLE
[DE-454] validation to update campaign content

### PR DESCRIPTION
**Pull request by Cromeror**

### CODE REVIEW

**Comment by andresmoschini** [comment]
Nice! Could you add a test for the validation error scenario?

File: Doppler.HtmlEditorApi/Controllers/CampaignsController.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@
 using Doppler.HtmlEditorApi.Model;
 using Doppler.HtmlEditorApi.Infrastructure;
 using System.Text.Json;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi/Controllers/CampaignsController.cs:11]: Could you revert this unnecessary change?

File: Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+            // Assert
+            Assert.Equal(expectedStatusCode, response.StatusCode);
+            Assert.Equal("Bearer", response.Headers.WwwAuthenticate.ToString());
+        }
+
+        [Theory]
+        [InlineData("/accounts/x@x.com/campaigns/123/content", TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518, HttpStatusCode.Forbidden)]
+        [InlineData("/accounts/x@x.com/campaigns/123/content", TOKEN_EXPIRE_20330518, HttpStatusCode.Forbidden)]
+        public async Task GET_campaign_should_not_accept_the_token_of_another_account(string url, string token, HttpStatusCode expectedStatusCode)
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs:55]: Is this test related to GET ok?

File: Doppler.HtmlEditorApi/Model/Campaign.cs [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
 
 namespace Doppler.HtmlEditorApi.Model;
 
-public record CampaignContent(JsonElement meta, string htmlContent) : Content(meta, htmlContent);
+public record CampaignContent(
+    [Required]
+    JsonElement meta,
+    [Required]
+    string htmlContent) : Content(meta, htmlContent), IValidatableObject
+{
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi/Model/Campaign.cs:13]:
```suggestion
    {
```

Comment (andresmoschini) [reply Doppler.HtmlEditorApi/Model/Campaign.cs:13]: Ooops

**Comment by andresmoschini** [comment]
❤️ 🚀

File: Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs (hunk 2 above)
Comment (Cromeror) [reply Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs]: Now its ok 💪
